docs/_config.yml

```diff
@@ -99,6 +99,8 @@ collections:
     output: true
     permalink: reference/mesheryctl/:collection/:path
     name: mesheryctlcommands
+  models:
+    output: true  
   # integrations:
```

Comment:
@vishalvivekm why are we not uncommenting `integrations` and using that collection?

Reply:
@leecalcote `_integrations` is already a directory which has adapters metadata with versioning :
<img width="153" alt="image" src="https://github.com/meshery/meshery/assets/110674407/1cd25c66-0406-4022-8743-b8de74dea418">

here's the link: 
https://github.com/meshery/meshery/tree/master/docs/_integrations

and yes `_adapters` is a collection dedicated to generate the pages.

Reply:
Not that integrations folder, but the integrations folder under extensibility, @vishalvivekm. You might need to cleanup where root of where `collections` are stored.